src/index.js

```diff
@@ -99,6 +100,7 @@ const weightedrandom_level_message = (level, percentSuffix = '') => {
   if (level === undefined) {
     return "The queue is empty.";
   }
+  twitch.notLurkingAnymore(level.submitter); // If we pull up a level, we should reset the lurking status
```

Comment:
`twitch.notLurkingAnymore(level.submitter);` would need to be replaced with `twitch.notLurkingAnymore(level.username);`.

Reply:
Resolved as requested.